[V8] Stylelint rule for removed `navds`-prefix (#4304)
navikt/Designsystemet: @navikt/aksel-stylelint/README.md

}
```

## aksel/no-legacy-classnames

> Warns when trying to to use legacy-classNames starting with `.navds`. In verson 8.0, all classNames now use the `.aksel`-prefix, so all old overrides will no longer work.

Reviewer: I think we should only write "className" if we are referring to the prop.

```suggestion
Warns when trying to to use legacy class names starting with `.navds`. As of version 8.0, all class names are prefixed with `.aksel`, so all old overrides no longer work.
```

> We still discourage overriding classNames from `@navikt/ds-css`:

Reviewer: ```suggestion
We still discourage overriding class names from `@navikt/ds-css`:
```

- Create a new className that you target not referencing the designsystem-css directly

> - Create an Github issue if what you need to do is not easily possible, and we can work together to find a solution.

Reviewer: ```suggestion
- Add your own `className` instead of referencing Aksel classes directly.
- Open a GitHub issue if what you need to do isn't straightforward, and we'll work together to find a solution.
```
